charts/csm-authorization-v2.0/charts/redis/templates/redis.yaml

```diff
@@ -106,26 +134,30 @@ spec:
           mountPath: /csm-auth-redis-cm/
         - name: config
           mountPath: /etc/redis/
-        {{- if .Values.redisSecretProviderClass.redisSecretName }}
-        - name: secrets-store-inline-redis
-          mountPath: /etc/csm-authorization/redis
+        {{- if and $secretProviderClasses (hasKey $secretProviderClasses "vault") (.Values.redisSecretProviderClass.redisSecretName) }}
+        {{- range $secretproviderclass := $secretProviderClasses.vault }}
```

Comment:
Is it possible that user configured multiple SecretProviderClasses for storage credentials, but configured only one of them with redis? In such case, should we mount all SPCs? is it not sufficient to mount that one redis related SPC?

Reply:
Updated the values so that we'll only get the SPC that holds redis creds